frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/networking-tab/networking-tab.tsx

```diff
@@ -127,9 +127,11 @@ const NetworkingTabComponent: React.FC<NetworkingTabComponentProps> = ({
             <Title headingLevel="h5" size="lg">
               No network interface added
             </Title>
-            <Button {...addButtonProps} icon={<PlusCircleIcon />} variant={ButtonVariant.link}>
-              {ADD_NETWORK_INTERFACE}
-            </Button>
+            {!isCreateDisabled && (
```

Comment:
we should include the buttons itself too (not just the empty flow)

Reply:
added